gensim/models/fasttext.py

```diff
@@ -745,7 +741,10 @@ def clear_sims(self):
         You can recompute them later again using the :meth:`~gensim.models.fasttext.FastText.init_sims` method.
 
         """
-        self._clear_post_train()
+        self.wv.vectors_norm = None
+        self.wv.vectors_vocab_norm = None
+        self.wv.vectors_ngrams_norm = None
+        self.wv.buckets_word = None
```

Comment:
This is fishy: nothing about training (as opposed to other vocab-discovery steps) would change the `buckets_word` mapping of words to ngram-locations. But also: (1) `init_sims()` doesn't rebuild this `buckets_word` as implied by the comment; (2) unless it's a big performance win in some usual case, I'm not sure why `buckets_word` needs to exist at all – I thought known-words already had presummed vectors (requiring no ngram lookups), while for any OOV word, it's not very expensive to find the ngrams/ngram-bucket-indexes each time needed; (3) relatedly, it looks like the only use of `vectors_ngrams_norm` is nonsensical, so that property should be discarded entirely - but I've made a separate issue, #2667, for that.